draft-ietf-quic-qpack.md

```diff
@@ -179,18 +179,25 @@ x ...
 QPACK instructions occur in three locations, each of which uses a separate
 instruction space:
 
- - Table updates are carried by a unidirectional stream from encoder to decoder.
-   Instructions on this stream modify the dynamic table state without generating
-   output to any particular request.
- - Acknowledgements of table modifications and header processing are carried by
-   a unidirectional stream from decoder to encoder.
+ - The encoder stream is a unidirectional stream of type `0x48` which carries
```

Comment:
H and D, not E and D?

Reply:
H and h, actually, for header compression.